app/views/users/confirmation.html.erb

```diff
@@ -1,20 +1,21 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts("Almost Done!") %></h2>
+<h2 class="heading"><%= t(".page_heading") %></h2>
 <!--/descriptions-->
 <!--main content-->
 <div class="userstuff">
   <p>
-    <%= ts("You should soon receive a confirmation email at the address you gave us. This will have the link you need to follow in order to activate your account and complete the account creation process. The confirmation email will come from <strong>%{return_address}</strong> -- you might want to add this to your address book to make sure you get the email.", return_address: ArchiveConfig.RETURN_ADDRESS).html_safe %>
+    <%= t(".receive_email_html", return_address: tag.strong(ArchiveConfig.RETURN_ADDRESS)) %>
   </p>
   <p>
-    <%= ts("If you haven't received this email within 24 hours, and you don't find our email in your spam filter or Social folder, please <a href='%{support_path}'>contact Support</a> for help.", support_path: new_feedback_report_path).html_safe %>
+    <%= t(".no_email_html", contact_support_link: (link_to t(".contact_support"), new_feedback_report_path)) %>
```

Comment:
**Nitpick:** we're definitely mixing styles in places, but it looks like the style below is more common (and is more closely aligned with the [coding standards](https://github.com/otwcode/otwarchive/wiki/Coding-Standards#general-code-formatting))
```suggestion
    <%= t(".no_email_html", contact_support_link: link_to(t(".contact_support"), new_feedback_report_path)) %>
```